### PR TITLE
Added support for HDR10 video playback on direct3d11

### PR DESCRIPTION
**Pull request by slouken**

### CODE REVIEW

File: src/render/direct3d11/D3D11_PixelShader_HDR10.h [hunk 1]
```diff
@@ -0,0 +1,515 @@
...
+endif 
+mov r1.w, l(1.000000)
+mul o0.xyzw, r1.xyzw, v2.xyzw
+ret 
+// Approximately 36 instruction slots used
+#endif
+
+const BYTE g_main[] =
+{
+     68,  88,  66,  67, 110, 205,
```
Comment (madebr) [inline src/render/direct3d11/D3D11_PixelShader_HDR10.h:100]: Does SDL also commit shader sources?

Comment (slouken) [reply src/render/direct3d11/D3D11_PixelShader_HDR10.h:100]: Yep, they’re in this commit.

**Comment by slouken** [comment]
> The generated `*.h` are `0755` and have dos line endings, like the prev patch..

Okay, I'll fix that in a future commit.

File: src/render/direct3d11/D3D11_PixelShader_HDR10.h (hunk 1 above)
Comment (madebr) [reply src/render/direct3d11/D3D11_PixelShader_HDR10.h:100]: Oops, I didn't see them.
Sorry about the noise.

**Pull request merged**